src/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,14 @@
+
+import React from 'react';
+
+import styles from './Checkbox.module.css'
+
+function Checkbox( {children, ...props}) {
+  return (
+<label className={styles.menu__name}><input className={styles.menu__checkbox} type="checkbox" name={props.name}/><div className={styles.menu__checkmark}><svg className={styles.rect} width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's add Prettier and code formatting will be the same everywhere.

Reply:
It could be better give names for elements in the context of the component. Why prefix is `menu` if it is used for example in `form`?